src/services/videoconference/index.js

```diff
@@ -43,319 +43,314 @@ const { ObjectId } = require('../../helper/compare');
 
 const CLIENT_HOST = Configuration.get('HOST');
 
-class VideoconferenceBaseService {
-	constructor(app) {
-		this.app = app;
-		this.docs = {};
+function throwOnValidationErrors(scopeId, scopeName, options = null) {
+	if (!idAndScopeNameAreValid({ scopeId, scopeName })) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['idAndScopeNameAreValid' was used before it was defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)